Dockerfile

```diff
@@ -12,6 +12,7 @@ RUN echo 'deb https://download.opensuse.org/repositories/devel:/tools:/criu/Debi
     && dpkg --add-architecture ppc64el \
     && apt-get update \
     && apt-get install -y --no-install-recommends \
+        astyle \
```

Comment:
Why?

Reply:
Not really sure, I think maybe it was needed when running on travis (yeah, the PR was open so long ago :D), removed that and golint and tests pass fine. I need to keep gperf, though